docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,374 @@
+    - Writing a software package.
+    - Documenting research (reports, scientific papers)
+    - Other major use cases (please specify).
+
+    For each of the items above, provide additional information related to:
+    - How frequently do you perform this task?
+        - Never.
+        - Every few months.
+        - Monthly.
+        - Weekly.
+        - Daily.
+    - To what degree does Jupyter meet your expectations for this?
+        - Does not apply.
+        - Never meets.
+        - Rarely meets.
+        - Sometimes meets.
+        - Often exceeds.
+        - Always exceeds.
+    - To what degree do alternative tools meet your expectations for this?
+        - Does not apply.
+        - Never meets.
+        - Rarely meets.
+        - Sometimes meets.
+        - Often exceeds.
+        - Always exceeds.
+
+### Data
+
+8. What data sources are you primarily working with in your role? (pick up to 3)
+    - 🖥️ My local file system (e.g. files and folder on local machine).
+    - File system (e.g. HPC, EBS/EFS, JupyterHub volumes).
+    - Cloud object storage (e.g. buckets, S3, Blob, GS).
+    - SQL (e.g. PostgreSQL, MySQL).
+    - SQL - embedded (e.g. SQLite).
+    - NoSQL - columnar store (e.g. Parquet, Arrow, HDFS, BigQuery).
+    - NoSQL - document store (e.g. MongoDB, Elasticsearch, DynamoDB).
+    - Graph database (e.g. Neo4j, TigerGraph).
+    - Time Series (e.g. InfluxDB).
+    - Pub/ sub (e.g. Apache Kafka, Druid).
+    - Key value (e.g. Redis, MemcacheDB).
+    - Google Sheets.
+    - ❗Industry or field specific APIs.
+    - Streaming.
+    - Other (please specify).
+
+9. What data formats are you mostly working with? (pick up to 3)
+    - Tabular (e.g. csv, spreadsheet, SQL tables, Parquet).
+    - Images.
+    - Tensors (e.g. manually handling PyTorch, Tensorflow inputs).
+    - Nested (e.g. JSON, NoSQL document).
+    - Hierarchical Data Format (e.g. HDF5 or similar).
+    - Time series.
+    - Text.
+    - Audio.
+    - Video.
+    - 3D/ CAD.
+    - Graph (e.g. nodes, edges).
+    - Spatial/ geographic (e.g. coordinates, GIS).
+    - Game/ reinforcement simulation.
+    - ❗Industry-specific file formats.
+    - Other (please specify)
+
+10. Do you experience these **problems with data** in Jupyter? (rate from scale of 0-4)
+    - No grid view for manipulating/ filtering dataframes and arrays.
+    - Can’t see a list of my current variables.
+    - Plaintext or environment variable management of database passwords/ keys/ secrets.
+    - Lost data during failure or restart of kernel/ server.
+    - Data is too big to fit into memory on my machine/ server.
+    - Poor MVC/ ORM integrations (e.g. Django, Flask).
+    - Managing database/ source connections and secrets.
+    - Other (please specify)
+    
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+11. What type of analysis are you running? (up to 4)
```

Comment:
Missing word.
```suggestion
11. What type of analysis are you running? (pick up to 4)
```

Reply:
Thank you. I just went through and made sure each one included 'pick'